homeassistant/components/zha/sensor.py

```diff
@@ -261,6 +266,7 @@ class Battery(Sensor):
     _attr_state_class: SensorStateClass = SensorStateClass.MEASUREMENT
     _attr_entity_category = EntityCategory.DIAGNOSTIC
     _attr_native_unit_of_measurement = PERCENTAGE
+    _attr_translation_key: str = "battery_remaining"
```

Comment:
I don't think this should have a dedicated translation key actually. It gets one from the device class or something IIRC.

Reply:
Yeah, just checked and it's already properly translated on my system.